ideascube/mediacenter/templates/mediacenter/document_detail.html

```diff
@@ -18,7 +18,7 @@
     {% elif document.preview %}
         <a href="{% media document 'original' %}"><img src="{% media document 'preview' %}" /></a>
     {% endif %}
-    <div class="text">{{ document.summary }}</div>
+    <div class="text">{{ document.summary|safe }}</div>
```

Comment:
> This is a problem indeed. But it is not related at all with js or tinyMCE.

Indeed, that's why I separated those two points in two paragraphs.

> As we do not check anything on django side

Except we used to: before this change (the specific line I'm commenting about) then when displaying the text it would get escaped. With your change it doesn't get escaped any more when displaying it, which means we might have to escape it when saving it?